docs/services/motion/_index.md

```diff
@@ -0,0 +1,466 @@
+
+// Move the gripper
+moved, err := motionService.Move(context.TODO(), goalPose, worldState, nil, nil)
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### MoveSingleComponent
+
+*`MoveSingleComponent` allows you to bypass Viam’s internal motion planning entirely, if desired, for a single component.*
+
+`MoveSingleComponent` looks similar to `Move` above but may result in radically different behavior.
+
+`MoveSingleComponent` translates the given destination into the frame of the specified component, and then calls `MoveToPosition` on that component to move it to the desired location.
+As of April 21, 2023, [arms](/components/arm/) are the only component supported by `MoveSingleComponent`.
+
+As the name of the method suggests, only the single component specified by `component_name` will actuate.
+
+An example of when this may be useful is if you have implemented your own custom arm model, and wish to use your own motion planning for it.
+Implement `MoveToPosition` on that arm using whatever method you desire to plan motion to the specified pose, and then use `MoveSingleComponent` to pass the destination in the frame of any other robot component.
+
+{{% alert title="Note" color="note" %}} <a id="move-vs-movetoposition">
+
+If this method is called with an arm that uses Viam’s motion planning on the backend, then this method is equivalent to using `robot.TransformPose` to transform the destination into the frame of the arm, and then calling [`MoveToPosition`](/components/arm/#movetoposition) on the arm directly.
+Note that `arm.MoveToPosition` does not use `world_state`, so collision checking and obstacle avoidance *will not* be performed.
+
+If you need collision checking and obstacle avoidance, use [`Move`](#move).
+
+{{% /alert %}}
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `component_name` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): The name of the component to move.
+  This component must support the `move_to_position` API call with a Pose.
+  As of April 21, 2023, [arm](/components/arm/) is the only component so supported.
+
+- `destination` ([PoseInFrame](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame)):
+  Describes where the `component_name` should end up.
+  Can be any pose, from the perspective of any component whose location is configured as a [`frame`](../frame-system/).
+  The `destination` will be converted into the frame of the component specified in `component_name` when passed to `move_to_position`.
+
+- `world_state` ([WorldState](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.WorldState)) (*optional*): Data structure specifying information about the world around the robot.
+  **See [note above](#move-vs-movetoposition); a custom implementation of `MoveToPosition` may or may not make use of `world_state`.**
```

Comment:
Should be more explicit here. There is no "may or may not", it is only "will not".

Reply:
Doesn't it depend on whether the user implements it such that it will?

Reply:
Is this better?
```suggestion
  **See [note above](#move-vs-movetoposition); a custom implementation of `MoveToPosition` will not make use of `world_state` unless you write an implementation that uses it.**
```

Reply:
We completely removed WorldState from the component API, so I guess it depends on the definition of "custom". If it's an entirely new interface, is it really a custom `MoveToPosition` implementation? I feel like it is not.